### DISCUSSION TITLE
Please add ability to view the list of issues where I have commented

### DISCUSSION THREAD

**Discussion by mailinglists35** [community/community#111226] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Thanks

**Comment by murat-yasar** [comment]
Type the following query into the search bar at the top: 

is:issue commenter:mailinglists35

**Reply by mailinglists35** [reply]
nice tip, thanks. I hope I will be able to remember it. maybe make it available as a drop down auto complete suggestion of the search bar when I type `comm` or something?

**Reply by murat-yasar** [reply]
You don't need to remember actually! After getting the search results, you can see a small bookmark sign on the top right of the results page, which states "save" next to it. You can click on it an save that search term for the future!

<img width="1151" alt="Save-A-Search" src="https://github.com/community/community/assets/127419303/bb42d5a6-7db3-499a-93cb-f766151ec760">

For example, if you save that search as "my-comments", next time you click on the search bar an type just "my", you'll see an option as a drop-down like: "saved:my-comments" under saved queries. 

<img width="1146" alt="Saved-Search" src="https://github.com/community/community/assets/127419303/3199c377-e089-4a6c-b038-1526879981ee">

Don't panic if you cannot see the expected result, just check the side menu and see the options like issue, users, commits, etc. You'll find the result there.


<img width="1139" alt="Filter-Menu" src="https://github.com/community/community/assets/127419303/ba917413-9aa0-46e6-9939-c4ddb11ba7d4">

And if you want to change your saved search terms or name etc, you can type saved: in the search bar and then click on the "Managed Saved Searches".

<img width="885" alt="Manage-Saved-Searches" src="https://github.com/community/community/assets/127419303/74518f2a-3873-4e55-b3c6-9f947cc2aea5">

**Comment by AMDphreak** [comment]
Still needed ? +1 from me.

You can already filter with `commenter:@me` in the search box, but the filter dropdowns show **Mentions** and hide **Commenter**. That inconsistency is especially painful when you want a shareable list of issues/PRs you actually engaged on (e.g. for hiring or handoff links).

I filed a related discussion that documents the UI gap and use cases: https://github.com/orgs/community/discussions/202870

Hope Product can surface Commenter (and Involves) next to the existing Mentions control.
